### PR TITLE
DO-1: refactor: do initialize testing

### PR DESCRIPTION
**Pull request by MilaKolasava**

### CODE REVIEW

File: src/components/Header/__spec__/header.spec.tsx [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+
+  expect(headerElement).toBeInTheDocument();
+});
+
+test("should render loupe", () => {
+  const setModalActive = jest.fn;
+  render(<Header setModalActive={setModalActive} />);
+
+  const headerElement = screen.getByTestId("header-loupe");
+  expect(headerElement).toBeInTheDocument();
```
Comment (glebpigulevsky) [inline src/components/Header/__spec__/header.spec.tsx:21]: empty line

File: package.json [hunk 2]
```diff
@@ -40,6 +39,7 @@
     ]
   },
   "devDependencies": {
+    "@testing-library/react": "^13.3.0",
```
Comment (glebpigulevsky) [inline package.json:42]: could you add script to run test command? like for npm start

File: src/components/ModalWindow/__spec__/modal.spec.tsx [hunk 3]
```diff
@@ -0,0 +1,23 @@
+import React from "react";
```
Comment (glebpigulevsky) [inline src/components/ModalWindow/__spec__/modal.spec.tsx]: why did you call file modal.*?

File: package.json (hunk 2 above)
Comment (MilaKolasava) [reply package.json:42]: Unfortunately I couldn't add script. 
I don't know how to do this.

File: src/components/ModalWindow/__spec__/modal.spec.tsx (hunk 3 above)
Comment (MilaKolasava) [reply src/components/ModalWindow/__spec__/modal.spec.tsx]: becouse it shorter then "modalWindow"
renamed fale

File: src/components/Header/__spec__/header.spec.tsx (hunk 1 above)
Comment (MilaKolasava) [reply src/components/Header/__spec__/header.spec.tsx:21]: done

File: src/components/ModalWindow/__spec__/modal.spec.tsx (hunk 3 above)
Comment (glebpigulevsky) [reply src/components/ModalWindow/__spec__/modal.spec.tsx]: fileName of the test should be same as components file name

**Review by glebpigulevsky — APPROVED** [review]

**Pull request merged**